RELEASE-NOTES.md

```diff
@@ -14,6 +14,7 @@
 * Empty strings are now detected as invalid language codes in the term classes
 * Made the `Item` constructor parameters optional
 * Made the `Fingerprint` constructor parameters optional
+* Made the `Property` constructor parameter for fingerprint nullable
```

Comment:
I know you are talking about the release notes, but what's "not very consistent" here?